reqs/tests/link_agencies_tests.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from model_mommy import mommy
+
+from reqs.management.commands import link_agencies
+from reqs.management.commands.sync_agencies import Command as SynchCommand
+from reqs.models import Agency, AgencyGroup, Requirement
+
+
+@pytest.fixture
+def populate_agencies():
+    synch = SynchCommand()
+    synch.create_system_groups()
+    synch.sync_row({
+        'agencyAbbreviation': None,
+        'agencyCode': '123',
+        'agencyName': 'Aquarius',
+        'agencyType': '5-Other Branches',
+        'CFO_Act': '1',
+        'CIO_Council': None,
+    })
+    synch.sync_row({
+        'agencyAbbreviation': "DOD",
+        'agencyCode': '111',
+        'agencyName': 'Department of Defense',
+        'agencyType': '',
+        'CFO_Act': '1',
+        'CIO_Council': None,
+    })
+    synch.sync_row({
+        'agencyAbbreviation': "DHS",
+        'agencyCode': '100',
+        'agencyName': 'Department of Homeland Security',
+        'agencyType': '',
+        'CFO_Act': '1',
+        'CIO_Council': None,
+    })
+    synch.sync_row({
+        'agencyAbbreviation': "GSA",
+        'agencyCode': '101',
+        'agencyName': 'General Services Administration',
+        'agencyType': '',
+        'CFO_Act': '1',
+        'CIO_Council': None,
+    })
+    synch.sync_row({
+        'agencyAbbreviation': "Justice",
+        'agencyCode': '102',
+        'agencyName': 'Department of Justice',
+        'agencyType': '',
+        'CFO_Act': '1',
+        'CIO_Council': None,
+    })
+    synch.sync_row({
+        'agencyAbbreviation': "FOO",
+        'agencyCode': '112',
+        'agencyName': 'Department of Foo',
+        'agencyType': '',
+        'CFO_Act': None,
+        'CIO_Council': None,
+    })
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("impacted_entity,agencies,agency_groups", [
+    (("",), (set(),), (set(),)),
```

Comment:
I think Python needs to give me a way to easily create empty tuples and lists with comprehensions.